examples/cactus-example-electricity-trade/tools/periodicExecuter/package.json

```diff
@@ -7,6 +7,12 @@
     "build-ts": "tsc",
     "tslint": "tslint -c tslint.json -p tsconfig.json"
   },
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Hyperledger Cactus Contributors",
```

Comment:
@micoferdinand98 Please update this to say `Cacti` instead of `Cactus`

Reply:
Roger that Peter